### PR TITLE
chore(picker): use new tokens

### PR DESCRIPTION
**Pull request by pfulton**
## Description

Updates the Picker to use the new tokens system & uses a beta release. Hold for full release

## Related issue(s)

-

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_
    1. Go here
    2. Do this

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ ] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ ] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Comment by pfulton** [comment]
CSS PR is [here](https://github.com/adobe/spectrum-css/pull/1528).

**Comment by Westbrook** [comment]
Is it possible to address https://github.com/adobe/spectrum-web-components/issues/2854 as part of this work?

**Comment by pfulton** [comment]
> Is it possible to address #2854 as part of this work?

Yeah, I can take a look at that!

File: packages/picker/src/spectrum-picker.css [hunk 1]
```diff
@@ -49,6 +49,11 @@ governing permissions and limitations under the License.
     --spectrum-picker-font-size: var(
         --spectrum-font-size-100
     ); /* .spectrum-Picker */
+    --spectrum-picker-placeholder-font-style-default: italic;
+    --spectrum-picker-placeholder-cjk-font-style: normal;
+    --spectrum-picker-placeholder-font-style: var(
+        --spectrum-picker-placeholder-font-style-default
+    );
```
Comment (pfulton) [inline packages/picker/src/spectrum-picker.css:56]: Added a few custom properties to handle the different `font-style`s.

File: packages/picker/src/spectrum-picker.css [hunk 2]
```diff
@@ -151,6 +156,15 @@ governing permissions and limitations under the License.
...
 }
+#button:lang(ja),
+#button:lang(ko),
+#button:lang(zh) {
+    --spectrum-picker-placeholder-font-style: var(
+        --spectrum-picker-placeholder-cjk-font-style
+    ); /* .spectrum-Picker:lang(ja),
+   * .spectrum-Picker:lang(zh),
+   * .spectrum-Picker:lang(ko) */
+}
```
Comment (pfulton) [inline packages/picker/src/spectrum-picker.css:167]: I _think_ this will work for y'all.

File: packages/picker/src/spectrum-picker.css [hunk 3]
```diff
@@ -832,7 +846,10 @@ governing permissions and limitations under the License.
             var(--spectrum-picker-font-color-default)
         )
     );
-    font-style: italic;
+    font-style: var(
+        --mod-picker-placeholder-font-style,
+        var(--spectrum-picker-placeholder-font-style)
+    );
```
Comment (pfulton) [inline packages/picker/src/spectrum-picker.css:852]: `font-style` in practice.

**Review by Westbrook — APPROVED** [review]
Looking good! Thanks, @pfulton 🙇🏼

**Pull request merged**